qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -409,6 +409,13 @@
             gagent_check_type = user_logoff
         - gagent_qgastatus_after_remove_qga:
             gagent_check_type = qgastatus_after_remove_qga
+        - gagent_container:
+            only Linux
+            no RHEL.8.0, RHEL.8.1
+            gagent_check_type = container
+            gagent_container_start_cmd = "podman start qemu-guest-agent"
+            get_target_ver_cmd = "cat /refer_ver.txt"
+            get_container_ver_cmd = "podman inspect qemu-guest-agent | grep %s"
```

Comment:
```
# Please define create_container_cmd before this test
create_container_cmd =

Reply:
@vivianQizhu okay, I added this comment line to .cfg.